posawesome/posawesome/api/qz.py

```
@@ -0,0 +1,176 @@
+        file.write(
+            key.private_bytes(
+                encoding=serialization.Encoding.PEM,
+                format=serialization.PrivateFormat.PKCS8,
+                encryption_algorithm=serialization.NoEncryption(),
+            )
+        )
+    try:
+        os.chmod(key_path, 0o600)
+    except Exception:
+        # Non-POSIX platforms may not support chmod modes the same way.
+        pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)